src/components/lists/items/CallAssignmentsListItem.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import LoadingIndicator from '../../misc/LoadingIndicator';
+import ParticipantList from './elements/ParticipantList'
+
+import {
+    retrieveCallAssignmentStats,
+    retrieveCallAssignmentCallers,
+} from '../../../actions/callAssignment';
+
+@connect(state => state.callAssignments)
```

Comment:
What's the purpose of this? As far as I can tell, no properties from state.callAssignments are used in the component. And even if they were, this way of mapping state to properties is a little bit weird.